src/modules/webhooks/stripe-webhook.ts

```diff
@@ -433,13 +433,12 @@ function stripePaymentIntentEventToPartialTransactionEventReportMutationVariable
   switch (stripeEvent.type) {
     // handling these is required
     case "payment_intent.succeeded": {
+      if (manualCapture) break;
```

Comment:
```suggestion
```

In [Stripe docs](https://docs.stripe.com/payments/paymentintents/lifecycle#intent-statuses) the `succeeded` status for PaymentIntent is sent only when funds are in the account, i.e. captured. The capture could have used manual capture method, but this doesn't matter for this event. I think we should just remove this condition.

![CleanShot 2024-02-19 at 16 31 42@2x](https://github.com/saleor/saleor-app-payment-stripe/assets/4144459/af73c1a2-6127-42d8-a29e-521232c2d381)

Reply:
I am not even sure that the CHARGE_REQUESTED would resolve the case, as the race condition still exists between wether the payment_intent.suceeded or saleor adds the transaction event first :)